Disher/View Controllers/LoginViewController.m

```diff
@@ -0,0 +1,98 @@
+//
+//  LoginViewController.m
+//  Disher
+//
+//  Created by Bienn Viquiera on 7/5/22.
+//
+
+#import "LoginViewController.h"
+@import Parse;
+
+
+@interface LoginViewController ()
+@property (weak, nonatomic) IBOutlet UITextField *usernameField;
+@property (weak, nonatomic) IBOutlet UITextField *passwordField;
+
+@end
+
+@implementation LoginViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    // Do any additional setup after loading the view.
+    
+}
+
+//BUTTON ACTIONS
+
+- (IBAction)didTapLogin:(id)sender {
+    [self loginUser];
+}
+
+- (IBAction)didTapSignup:(id)sender {
+    [self registerUser];
+}
+
+//HELPER METHODS
+
+- (void) loginUser {
+    NSString *username = self.usernameField.text;
+    NSString *password = self.passwordField.text;
```

Comment:
should we perform any checks/validations for these fields? what happens if we send nil to the Parse API?